Cubical/Structures/Group/Base.agda

```diff
@@ -242,3 +240,16 @@ is-set (isSemigroup (isMonoid (isGroup intGroup))) = isSetInt
 assoc (isSemigroup (isMonoid (isGroup intGroup))) = +-assoc
 identity (isMonoid (isGroup intGroup)) x = refl , (+-comm (pos 0) x)
 inverse (isGroup intGroup) x = +-comm x (pos 0 -Int x) ∙ minusPlus x 0 , (minusPlus x 0)
+
+
+dirProd' : Group₀ → Group₀ → Group₀
```

Comment:
What is this? Why '? Please add a comment

Reply:
Oh wups, that's some silly exeperiment that I forgot to delete